config/crd/bases/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -305,6 +305,49 @@ spec:
                         pattern: ^(Managed|Unmanaged|Force|Removed)$
                         type: string
                     type: object
+                  trustyai:
+                    description: TrustyAI component configuration.
+                    properties:
+                      devFlags:
+                        description: Add developer fields
+                        properties:
+                          manifests:
+                            description: List of custom manifests for the given component
+                            items:
+                              properties:
+                                contextDir:
+                                  default: ""
+                                  description: contextDir is the relative path to
+                                    the folder containing manifests in a repository
+                                  type: string
+                                sourcePath:
+                                  default: ""
+                                  description: 'sourcePath is the subpath within contextDir
+                                    where kustomize builds start. Examples include
+                                    any sub-folder or path: `base`, `overlays/dev`,
+                                    `default`, `odh` etc'
+                                  type: string
+                                uri:
+                                  default: ""
+                                  description: uri is the URI point to a git repo
```

Comment:
Wondering why you call it `git repo` if it looks like just a tarball without any git leftovers.

Reply:
@ykaliuta This seems to be the default Kubebuilder comment from https://github.com/opendatahub-io/opendatahub-operator/blob/5bc8caf52b1aa8f141169f37ff767fa36bbfeb32/components/component.go#L42